push/Push.cpp

```diff
@@ -226,7 +226,7 @@ std::string CPush::ProcessSendValue(const std::string &rawsendValue, const int d
 	}	
 	else if (vType == "Counter")
 	{
-		sprintf(szData,"Not supported yet");
+		strcpy(szData, rawsendValue.c_str())
```

Comment:
I don't think you tested this code ;)
(See our automatic build system)
Please always test...